mapie/time_series_regression.py

```diff
@@ -255,7 +264,8 @@ def predict(
         self.lower_quantiles_ = lower_quantiles
         self.higher_quantiles_ = higher_quantiles
 
-        if self.cv == "prefit":
+        if self.method in self.no_agg_methods_ \
+                or self.cv in self.no_agg_cv_:
```

Comment:
no_agg_cv includes split, is it implemented here ?

Reply:
It inherits from `MapieRegressor`. So yes, it is implemented.